src/Peregrine/__tests__/Peregrine.test.js

```diff
@@ -11,7 +11,12 @@ configure({ adapter: new Adapter() });
 
 beforeAll(() => {
```

Comment:
This should be `beforeEach` and `afterEach`. Otherwise it's too easy for `toHaveBeenCalled` type assertions to pass because of state left around in previous tests

Reply:
Not my PR, but I agree!